geopandas/plotting.py

```diff
@@ -137,11 +138,20 @@ def plot_series(s, cmap='Set1', color=None, ax=None, linewidth=1.0,
         else:
             col = color
         if geom.type == 'Polygon' or geom.type == 'MultiPolygon':
-            plot_multipolygon(ax, geom, facecolor=col, linewidth=linewidth, **color_kwds)
+            if 'facecolor' in color_kwds:
+                plot_multipolygon(ax, geom, linewidth=linewidth, **color_kwds)
+            else:
+                plot_multipolygon(ax, geom, facecolor=col, linewidth=linewidth, **color_kwds)
         elif geom.type == 'LineString' or geom.type == 'MultiLineString':
-            plot_multilinestring(ax, geom, color=col, linewidth=linewidth)
+            if 'color' in color_kwds:
+                plot_multilinestring(ax, geom, linewidth=linewidth, **color_kwds)
+            else:
+                plot_multilinestring(ax, geom, color=col, linewidth=linewidth, **color_kwds)
```

Comment:
This one is already handled above I think (there is a check `if color is None` to get the correct color), so the if/else is not needed here I think